PROTO_tests/tests/operator_test.py

```diff
@@ -0,0 +1,774 @@
+import warnings
+from itertools import product
+
+import numpy as np
+import pytest
+
+import arkouda as ak
+
+NUMERIC_TYPES = ["int64", "float64", "bool", "uint64"]
+
+
+def make_np_arrays(size, dtype):
+    if dtype == "int64":
+        return np.random.randint(-(2**32), 2**32, size=size, dtype=dtype)
+    elif dtype == "uint64":
+        return np.random.randint(-(2**32), 2**32, size=size).astype(dtype)
+    elif dtype == "float64":
+        return np.random.uniform(-(2**32), 2**32, size=size)
+    elif dtype == "bool":
+        return np.random.randint(0, 1, size=size, dtype=dtype)
+    return None
+
+
+class TestOperator:
+    def test_numpy_equivalency(self, size=100, verbose=pytest.verbose):
+        # ignore numpy warnings like divide by 0
+        np.seterr(all="ignore")
+        global pdarrays
```

Comment:
Maybe I am missing something, but I don't think we need the global definitions here. Was there a reason we made these global definitions?

Reply:
to be honest, this was just leftover from how it used to be in `run_test` where that lived outside the class. I think you're right that it doesn't need to be that way anymore. I'll update this

Reply:
Upon trying to delete the globals, I found that `do_op` requires these to be globals in order for the `eval` to work properly
```python
def do_op(lt, rt, ls, rs, isarkouda, oper):
    evalstr = ""
    if ls:
        evalstr += f'scalars["{lt}"]'
    else:
        evalstr += f'{("ndarrays", "pdarrays")[isarkouda]}["{lt}"]'
    evalstr += f" {oper} "
    if rs:
        evalstr += f'scalars["{rt}"]'
    else:
        evalstr += f'{("ndarrays", "pdarrays")[isarkouda]}["{rt}"]'
    res = eval(evalstr)
    return res
```

Reply:
Ah ok. Sounds good. As long as there is a reason for them.